pyta.py

```diff
@@ -10,24 +10,25 @@
 """
 import importlib.util
 import pylint.lint as lint
-from pylint.reporters import BaseReporter
+from reporters.color_reporter import ColorReporter
 
 import webbrowser
 import os
 from urllib.request import pathname2url
 
+
 # Local version of website; will be updated later.
 HELP_URL = 'file:' + pathname2url(os.path.abspath('website/index.html'))
 
 
-def check(module_name):
+def check(module_name, reporter=ColorReporter()):
```

Comment:
@shweta97 we resolved this in the meeting last week. You should update your pull request accordingly (and do another pull from master to resolve conflicts).